content/geometry/object-vehicle/vehicle-naming-convention.adoc

```diff
@@ -0,0 +1,209 @@
+
+| *Low_Beam*
+| Specifies low beam lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *High_Beam*
+| Specifies high beam lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Corner*
+| Specifies corner lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Position*
+| Specifies position lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Park*
+| Specifies park lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Indicator*
+| Specifies indicator lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *License_Plate*
+| Specifies license plate lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Tail*
+| Specifies tail lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Brake*
+| Specifies brake lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Reverse*
+| Specifies reverse lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Fog*
+| Specifies fog lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *Warning*
+| Specifies warning lights of the vehicle and is always used after the subkey "Light".
+  After this subkey always follows a subkey which indicates the specific view side.
+
+|===
+
+.Mirror Subkeys
+[%header, cols="20, 80"]
+|===
+
+2+^| Vehicle Mirror Subkeys
+
+| *Side*
+| Specifies the side mirror(s) of an vehicle.
+
+| *Blindspot*
+| Specifies the blindspot mirror(s) of an vehicle
+
+| *Rearview*
+| Specifies the rearview mirror(s) of an vehicle
+
+| *Mounting*
+| Specifies the mirror mounting on the vehicle, always used after the subkey, which specifies the type of the mirror
+
+|*Joint*
+| Specifies the movable part of the mirror on the mirror mounting.
+  After this subkey always follows a subkey which indicates the specific view side.
+
+| *View*
+| Specifies the view direction of the mirror glass on a the mirror
+
+|===
+
+.Additional and recommended Subkeys
```

Comment:
Should we make this part of the standard for release 1.0?
We postponed standardization of vehicle_parts...